utilities/post_migration.py

```
@@ -214,6 +215,102 @@ def _read_marker() -> str | None:
+        vm_name (str): Name of the migrated VM.
+        vm_ssh_connections (SSHConnectionManager): SSH connection manager.
+        source_provider_data (dict[str, Any]): Provider configuration with guest credentials.
+        source_vm_info (dict[str, Any]): VM information including OS type.
+        timeout (int): Maximum seconds to wait for SSH connectivity.
+        retry_delay (int): Seconds between SSH retry attempts.
+
+    Raises:
+        AssertionError: If no LUKS-encrypted devices are found.
+        TimeoutExpiredError: If SSH connectivity cannot be established.
+    """
+    LOGGER.info(f"Verifying LUKS encryption on migrated VM {vm_name}")
```

Ruff (0.15.17)

[warning] Logging statement uses f-string

(G004)